.github/workflows/sentry_delayed_job_test.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       env:
         RUBYOPT: ${{ matrix.options.rubyopt }}
       run: |
+        [[ "${{ matrix.ruby_version }}" =~ ^(2.4|2.5)$ ]] && bundle config force_ruby_platform true
```

Comment:
TIL this config. What does it do?

Reply:
The latest [sqlite3 release](https://github.com/sparklemotion/sqlite3-ruby/releases/tag/v1.5.0) only publishes binaries from ruby 2.6 onwards and CI was failing on 2.4/2.5 because bundler was trying to look for binaries instead of installing the ruby platform version (compiling native extensions). 

This config forces bundler to always compile on 2.4 and 2.5.

Reply:
ok this is no longer needed with latest bundler, removed